packages/2d/src/components/Line.ts

```diff
@@ -251,10 +263,18 @@ export class Line extends Shape {
 
     context.save();
     context.beginPath();
-    if (this.endArrow()) {
+    if (this.endArrow() instanceof Node) {
+      const endArrow = this.endArrow() as Node;
+      endArrow.position(endPoint);
```

Comment:
Editing signals inside of `render` / `draw` may cause unwanted behavior.
The `render` method is not guaranteed to be called each time the main generator progresses.
This means that accessing the position of this node from within a generator will return random values, depending on whether `render` was called or not.

Instead, you should transform the context to put the node in the correct location.
You should also rotate the matrix because right now arrows will be oriented the same way.

Reply:
yes makes sense. I forgot about the rotation sorry...